Lib/_pyrepl/readline.py

```diff
@@ -38,7 +38,14 @@
 
 from . import commands, historical_reader
 from .completing_reader import CompletingReader
-from .unix_console import UnixConsole, _error
+from .console import Console as ConsoleType
+
+Console: type[ConsoleType]
+_error: tuple[type[Exception], ...] | type[Exception]
+try:
+    from .unix_console import UnixConsole as Console, _error
+except:
```

Comment:
Is it possible to catch a more specific error here, e.g. `ImportError`? I worry that users might get a long and confusing traceback if the `try` and the `except` branches both fail, but maybe that can't be help